docs/ops/roundtable/index.rst

```diff
@@ -26,14 +26,22 @@ It deploys:
 
   - :doc:`argo-cd <../argo-cd/index>` for continuous delivery of Roundtable apps with Argo CD.
   - :doc:`prometheus <../prometheus/index>` for the Kubernetes-native monitoring stack (Prometheus Operator, Prometheus, and Grafana).
+  - :doc:`vault-secrets-operator <../vault-secrets-operator/index>` to retrieve secrets from Vault and store them as Kubernetes secrets.
 
 This app depends on the :doc:`security <../security/index>` app, which provides secret management facilities and ingress.
 
 .. rubric:: Bootstrapping the Application
 
 Since ``roundtable`` is a parent app, its ``Application`` resource was not created automatically and is not managed by GitOps.
 
-We manually created the ``roundtable`` ``Application`` from the :command:`argocd` CLI:
+Before bootstrapping the ``roundtable`` app, the ``security`` app needs to be bootstrapped so that Vault is running.
+See :doc:`its documentation <../security/index>` for more information.
+Then, a Vault access token for the Vault Secrets Operator must be created in Vault and stored as the ``vault-secrets-operator`` Kubernetes secret with two keys:
```

Comment:
This means we're manually deploying the `vault-secrets-operator` Secret? I think this section of the doc could be improved by replacing the list of keys with an actual YAML code block showing a template of the Secret, including namespace and keys (of course `...` over the values :) ). I think that would make it just that bit easier for another person to reproduce the step.